test/account_api/account_api_test.rb

```diff
@@ -5,8 +5,7 @@
 describe GdsApi::AccountApi do
   include GdsApi::TestHelpers::AccountApi
 
-  let(:base_url)      { Plek.find("account-api") }
-  let(:api_client)    { GdsApi::AccountApi.new(base_url) }
+  let(:api_client) { GdsApi::AccountApi.new(Plek.find("account-api")) }
```

Comment:
Do we need the Plek line here?
I've been l[ooking through the GDS API](https://github.com/alphagov/gds-api-adapters/blob/603fb33f3894559029728eb9143d1644a7cd858e/lib/gds_api.rb#L32-L35)

Looked a bit like we could call `GdsApi.account_api` which also fetches stuff out of plek?

Reply:
Yeah, I did this to be consistent with the other tests